.github/workflows/build_and_publish.yaml

```diff
@@ -75,7 +75,7 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v1
     - name: set VERSION environment variable
-      run: echo "VERSION='${{github.event.inputs.tag || needs.tag-new-version.outputs.tag}}'" >> $GITHUB_ENV
+      run: echo "VERSION=v${{github.event.inputs.tag || needs.tag-new-version.outputs.tag}}" >> $GITHUB_ENV
```

Comment:
Does `v` do anything here? Or is it just the literal string `v`?

Reply:
Just a literal v as that's been our convention for tag names